tsconfig.json

```diff
@@ -16,7 +16,10 @@
     "noImplicitAny": true,
     "noImplicitReturns": true,
     "noUnusedLocals": true,
-    "noUnusedParameters": true
+    "noUnusedParameters": true,
+
+    /* Temporary workaround until @ar.io/sdk is fixed */
+    "skipLibCheck": true
```

Comment:
@djwhitt are you good with that?

Reply:
the issue comes from warp-contracts and `@irys/arweave` - I've pushed an issue to warp but don't expect them to handle immediately. have not diagnosed the @irys/arweave issue yet

Reply:
Yeah, this is fine for now.